File: C,C++/C++/Exercicios/menor_de_tres.cpp
#include <bits/stdc++.h>
using namespace std;

int main(){

    int N,i,menor;

    cout << "Quantos numeros serao digitados: ";
    cin >> N;

    int vet[N];

    for (i = 0; i < N; i++){
        cout << "Digite um numero: ";
        cin >> vet[i];
    }
    
    menor = vet[1];

    for (i = 0; i < N; i++){
        if (vet[i] < menor){
            menor = vet[i];
        }
    }
    
    cout << "MENOR = " << menor;

    return 0;
}

File: Python/Execicios/menor_de_tres.py
N = int(input("QUantos numeros serao digitados: "))

vet: [float] = [0 for x in range(N)]

for i in range(0,N):
    vet[i]= float(input("Digite um valor: "))


menor = vet[0]
for i in range(0,N):
    if vet[i] < menor:
        menor = vet[i]

print(f"MENOR = {menor}")